dotCMS/src/integration-test/java/com/dotmarketing/cms/urlmap/URLMapAPIImplTest.java

```diff
@@ -90,6 +91,133 @@ public void prepareTest() {
         urlMapAPI = new URLMapAPIImpl();
     }
 
+    /**
+     * UrlMapped Content that lives on the system host should be available on every host where the map
+     * detail page exists (by path). If content does not live on the system host, it should only be
+     * available on the host it is currently one
+     * 
+     * @throws Exception
+     */
+
+
+    @Test
+    public void systemHost_UrlMaps_Should_Work_Across_Hosts() throws Exception {
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5796110)